src/actions/onboarding.ts

```
@@ -197,3 +380,142 @@ export async function submitOnboarding(formData: FormData) {
+
+    return { success: true };
+  } catch (error) {
+    console.error("Error finalizing investor onboarding:", error);
+    throw new Error(
+      error instanceof Error ? error.message : "Failed to finalize onboarding"
+    );
+  }
+
+  // Revalidate and redirect outside of try-catch to avoid catching redirect errors
+  revalidatePath("/home");
+  redirect("/home");
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)